### PR TITLE
Update `How to build runtime` document

### PR DESCRIPTION
**Pull request by lemmaa**
issue #246

- Update, rewrite `how-to-build-runtime.md`
- Add empty documents for links for future creation
- update `index.rst`

Signed-off-by: Sung-Jae Lee <sj925.lee@samsung.com>

### CODE REVIEW

File: docs/howto/how-to-build-runtime.md [hunk 1]
```diff
@@ -1,9 +1,226 @@
...
+│   ├── test_onert_frontend_nnapi
+│   └── tflite_test
+└── unittest_standalone
+    └── nnfw_api_gtest
+
+19 directories, 36 files
+
+```
+
+Here, let's recall that the main target of our project is the arm architecture. If you have a development environment running Linux for arm on a device made of an arm CPU, such as Odroid-XU4, you will get a runtime binary that can be run on the arm architecture with the same command above. This is the simplest way to get a binary for an arm device. However, in most cases, native builds on arm devices are too impractical as they require too long. Therefore, we will create an executable binary of an architecture other than x86_64 through a `cross build`. For cross-build method for each architecture, please refer to the corresponding document in the following section, [How to cross-build runtime for different architevture](#how-to-cross-build-runtime-for-different-architevture).
```
Comment (dev-hue) [inline docs/howto/how-to-build-runtime.md:160]: [How to cross-build runtime for different architevture](#how-to-cross-build-runtime-for-different-architevture).
architevutre looks like typo.

Comment (lemmaa) [reply docs/howto/how-to-build-runtime.md:160]: @dev-hue , Thank you! Would you please review with diff suggestion for easy fix? :)

Comment (dev-hue) [reply docs/howto/how-to-build-runtime.md]: @lemmaa I didn't know that! Very useful! Thanks!
But, I miss-clicked the *commit suggestion* 😢.
I googled the way to revert it, I can't find...
Is there any way to revert..?

File: docs/howto/how-to-build-runtime.md [hunk 2]
```diff
@@ -157,7 +157,7 @@ $ tree -L 3 ./Product/out
...
+Here, let's recall that the main target of our project is the arm architecture. If you have a development environment running Linux for arm on a device made of an arm CPU, such as Odroid-XU4, you will get a runtime binary that can be run on the arm architecture with the same command above. This is the simplest way to get a binary for an arm device. However, in most cases, native builds on arm devices are too impractical as they require too long. Therefore, we will create an executable binary of an architecture other than x86_64 through a `cross build`. For cross-build method for each architecture, please refer to the corresponding document in the following section, [How to cross-build runtime for different architecture](#how-to-cross-build-runtime-for-different-architecture).
```
Comment (lemmaa) [inline docs/howto/how-to-build-runtime.md:160]: @dev-hue , I don't mean that make this commit from you. Just get know you the usage of '+-' button on toolbar menu of review comment window. If you click the button '+-', you can create a review comment with diff suggestion for fix, and I can accept it by single button click. Please experience with this review comment for next use. :)

** Please don't accept this diff suggestion, It's made for example. Please reject this suggestion if you read this comment.**

```suggestion
Here, let's recall that the main target of our project is the arm architecture. If you have a development environment running Linux for arm on a device made of an arm CPU, such as Odroid-XU4, you will get a runtime binary that can be run on the arm architecture with the same command above. This is the simplest way to get a binary for an arm device. However, in most cases, native builds on arm devices are too impractical as they require too long. Therefore, we will create an executable binary of an architecture other than x86_64 through a `cross build`. For cross-build method for each architecture, please refer to the corresponding document in the following section, [How to cross-build runtime for different architecture](#how-to-cross-build-runtime-for-different-architevture).
```

Comment (dev-hue) [reply docs/howto/how-to-build-runtime.md:160]: @lemmaa I understood perfectly! 👍 
But I couldn't find **reject button**.
There are just **Commit suggestion** and **Add suggestion to batch**.
It seems that I'm not the author.
If not, I'm sorry, please let me know how to reject. 😢

Comment (lemmaa) [reply docs/howto/how-to-build-runtime.md:160]: > .. But I couldn't find reject button. ...

@dev-hue , Yes, it's not there. Just ignore this comment and give 'approval' on PR if you agree. :)

Comment (lemmaa) [reply docs/howto/how-to-build-runtime.md:160]: I found a good guide to this feature. https://haacked.com/archive/2019/06/03/suggested-changes/

Comment (dev-hue) [reply docs/howto/how-to-build-runtime.md:160]: @lemmaa Thanks for many things!
I learned a lot today :)

**Pull request merged**